### PR TITLE
docs(docusaurus/): documentation in docusaurus/

### PR DESCRIPTION
**Pull request by Yalyalieva**
I put the documentation of the data analysis part to docusaurus/docs/data_analysis

### CODE REVIEW

File: docusaurus/docs/data_analysis/data_access/query_data.md
Comment (mariaBer) [inline docusaurus/docs/data_analysis/data_access/query_data.md:1]: Some places you wrote: "Show the output:" , "the result", and some "Output table" any reason for the difference, if not
Please change them to "Show the output"  / "Show the result"

**Review by mariaBer — CHANGES_REQUESTED** [review]
If I remember correctly we had a discussion about the documentation folder and if it should be under each category in data analysis or not, However I don't remember why the conclusion was to leave it in the way it is now.
Is it possible that thee documentation directory will be under each category rather than in documentation we will have again access, error etc.?

**Comment by mariaBer** [comment]
@Yalyalieva, did you see the review?

**Comment by Yalyalieva** [comment]
> @Yalyalieva, did you see the review?

Hi, @mariaBer , yes I saw the review and will work on it soon

**Comment by Yalyalieva** [comment]
> If I remember correctly we had a discussion about the documentation folder and if it should be under each category in data analysis or not, However I don't remember why the conclusion was to leave it in the way it is now. Is it possible that thee documentation directory will be under each category rather than in documentation we will have again access, error etc.?

The documentation is made automatically and if there are references in methods to other classes they are also written automatically. It is the reasoin why documentation must repeate the structure of the package, otherwise the references will not work

File: docusaurus/docs/data_analysis/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 1
+sidebar_label: 'CITROS Data Analysis'
+hide_title: true
+---
+# Data Analysis Overview
+
+This Python package is a comprehensive solution for data query, analysis and visualization. It is designed to work with data from your simulations, uploaded in PostgreSQL database. With its extensive features, you can quickly and easily extract valuable insights from your data.
+
+## Table of Contents
```
Comment (mariaBer) [inline docusaurus/docs/data_analysis/index.md:10]: Please change to see two levels rather than all of them

**Pull request merged**